### ISSUE TITLE
Author's notes no longer appear in ebook

### ISSUE THREAD

**Issue by MichaelBlume** [JasonGross/unsong_scraper#5]
#2 caused the build to use the TOC instead of next links, but author's notes don't appear in the TOC. *Links* to author's notes still appear at the end of relevant chapters, and the reader can follow them and read them in their browser, so that's probably ok? But it seemed worth mentioning.

**Comment by JasonGross** [comment]
Yeah, I guess the ideal thing would be to put them in some sort of appendix and set the links to point to the appendix?  I'm not inclined to put the effort in to do this, but thanks for bringing it up, and pull requests welcome.
